compiler/cpp/test/plugin/conversion_test.cc

```diff
@@ -269,6 +268,8 @@ void test_const_value(t_const_value* sut) {
 #undef T_CONST_VALUE_CASE
   case t_const_value::CV_MAP:
     BOOST_CHECK_EQUAL(sut->get_map().size(), sut2->get_map().size());
+    // NOTE This test only works with map with a single entry because we can't
+    // rely on map ordering with pointers as key.
     BOOST_CHECK_EQUAL(sut->get_map().begin()->first->get_type(),
                       sut2->get_map().begin()->first->get_type());
     BOOST_CHECK_EQUAL(sut->get_map().begin()->second->get_type(),
```

Comment:
I think we still want to have multiple values inside map, e.g. by using `operator[]`/`find` on sut2.